### ISSUE TITLE
3box@1.21.0 package has a problem with 3id-connect

### ISSUE THREAD

**Issue by dkent600** [3box/3box-js#889]
After upgrading to 3box@1.21.0 and then when running webpack, I get the error:

> ERROR in ./node_modules/3id-connect/lib/threeIdConnectService.js
Module not found: Error: Can't resolve 'identity-wallet' in '[...]\node_modules\3id-connect\lib'
 @ ./node_modules/3id-connect/lib/threeIdConnectService.js 24:21-47
 @ ./node_modules/3id-connect/lib/index.js
 @ ./node_modules/3box/lib/3box.js
 @ ./src/actions/profilesActions.ts
 @ ./src/layouts/AppContainer.tsx
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi react-hot-loader/patch webpack-dev-server/client?http://127.0.0.1:3000 webpack/hot/only-dev-server ./src/index.tsx

1.21.0 is thus not usable for me out-of-the-box.

**Comment by oed** [comment]
Thanks for reporting @dkent600 Looking into it!

**Comment by oed** [comment]
@dkent600 What version if identity-wallet gets installed?
Output of this command:
```
$ npm list identity-wallet
```

**Comment by pepoospina** [comment]
Hi @dkent600 @oed 

Same/similar error here. 

It happened after rebuilding our package-lock.json files, without updating the 3box version we have installed (which is `1.19.0`).

output of `npm list identity-wallet`

```
@uprtcl/evees@0.0.7-alpha.41 /home/pepo/pr/uprtcl/js-uprtcl-core/modules/evees
└─┬ 3box@1.21.0
  └─┬ 3id-connect@0.1.0
    └── identity-wallet@1.4.0 
```

**Comment by oed** [comment]
Looks like the ES5 build of `identity-wallet` wasn't being produced correctly. Should be fixed in `identity-wallet@1.4.1`.

**Comment by dkent600** [comment]
> Looks like the ES5 build of `identity-wallet` wasn't being produced correctly. Should be fixed in `identity-wallet@1.4.1`.

That works, thanks!
